components/infobox/wikis/splatoon/infobox_league_custom.lua

```diff
@@ -0,0 +1,98 @@
+---
+-- @Liquipedia
+-- wiki=splatoon
+-- page=Module:Infobox/League/Custom
+--
+-- Please see https://github.com/Liquipedia/Lua-Modules to contribute
+--
+
+local Class = require('Module:Class')
+local Lua = require('Module:Lua')
+local Logic = require('Module:Logic')
+local String = require('Module:StringUtils')
+local Variables = require('Module:Variables')
+
+local Injector = Lua.import('Module:Infobox/Widget/Injector', {requireDevIfEnabled = true})
+local League = Lua.import('Module:Infobox/League', {requireDevIfEnabled = true})
+
+local Widgets = require('Module:Infobox/Widget/All')
+local Cell = Widgets.Cell
+local Title = Widgets.Title
+
+local CustomLeague = Class.new()
+local CustomInjector = Class.new(Injector)
+
+local _args
+local _game
+
+local _GAME = mw.loadData('Module:GameVersion')
+
+function CustomLeague.run(frame)
+	local league = League(frame)
+	_args = league.args
+
+	league.addToLpdb = CustomLeague.addToLpdb
+	league.createWidgetInjector = CustomLeague.createWidgetInjector
+	league.defineCustomPageVariables = CustomLeague.defineCustomPageVariables
+	league.liquipediaTierHighlighted = CustomLeague.liquipediaTierHighlighted
+
+	return league:createInfobox(frame)
+end
+
+function CustomLeague:createWidgetInjector()
+	return CustomInjector()
+end
+
+function CustomInjector:parse(id, widgets)
+	if id == 'sponsors' then
+		table.insert(widgets, Cell{name = 'Official Device', content = {_args.device}})
+	elseif id == 'gamesettings' then
+		return {
+			Cell{name = 'Game version', content = {
+					CustomLeague._getGameVersion()
+				}},
+			}
+	elseif id == 'customcontent' then
+		if _args.player_number then
+			table.insert(widgets, Title{name = 'Players'})
+			table.insert(widgets, Cell{name = 'Number of players', content = {_args.player_number}})
+		end
+
+		--teams section
+		if _args.team_number then
+			table.insert(widgets, Title{name = 'Teams'})
+			table.insert(widgets, Cell{name = 'Number of teams', content = {_args.team_number}})
+		end
```

Comment:
In my personal opinion, skip the title

Reply:
Should I removed all these or just the **widgets, Title** line? 

(If I remove just **widgets, Title** the column are just gone) 
![image](https://user-images.githubusercontent.com/88981446/195319199-9aa00606-d6df-43a2-bcef-a2f3be22043a.png)

Reply:
Move it to `addCustomCells` instead of `customcontent`
https://github.com/Liquipedia/Lua-Modules/blob/af9000af5c64c211565cdec2cdf67636b87e5638/components/infobox/wikis/rainbowsix/infobox_league_custom.lua#L79-L99